### PR TITLE
Expose currentPlaybackTime when live stream video

### PR DESCRIPTION
**Pull request by paramaggarwal**
Features:

- Support `preferredForwardBufferDuration` prop on iOS
- Expose `currentPlaybackTime` on Android ExoPlayer and iOS
- Expose `trackId` on Android ExoPlayer

### CODE REVIEW

**Review by rishu-curefit — APPROVED** [review]

**Comment by paramaggarwal** [comment]
cc: @cobarx we have been using these changes at scale and I think the community will find them useful.

**Review by jensellfors — APPROVED** [review]
This looks good too me, I love the cross platform support :)

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java [hunk 1]
```diff
@@ -858,12 +873,14 @@ else if (e.type == ExoPlaybackException.TYPE_SOURCE) {
     }
 
     private static boolean isBehindLiveWindow(ExoPlaybackException e) {
+        com.google.android.exoplayer2.util.Log.e("ExoPlayer Exception", e.toString());
```
Comment (benoitdion) [inline android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java]: any reason you're using Log from exoplayer?

Comment (paramaggarwal) [reply android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java]: No particular reason actually. Should I change it to `Log.e`?

**Comment by paramaggarwal** [comment]
Hi, is there anything pending on me? Should I change that log statement? Let me know, thanks.

**Comment by jensellfors** [comment]
@paramaggarwal looks fine be me, no idea how Android logging works ;p. Can you also update README with this new prop. Add a line to CHANGELOG with link to PR and short explanation.

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java (hunk 1 above)
Comment (paramaggarwal) [reply android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java]: Changed to `Log.e()` 👍

**Comment by paramaggarwal** [comment]
I have made the requested changes @jenshandersson - thank you so much! 😊

**Comment by paramaggarwal** [comment]
I have made the requested changes @jenshandersson - thank you so much! 😊

**Pull request merged**